Stripe.xcodeproj/project.pbxproj

```diff
@@ -374,6 +374,23 @@
 		04F94DD31D22A23F004FC826 /* NSBundle+Stripe_AppName.h in Headers */ = {isa = PBXBuildFile; fileRef = 049A3F971CC76A2400F57DE7 /* NSBundle+Stripe_AppName.h */; };
```

Comment:
Changes in `project.pbxproj`:

- Added a bunch of test files

- Added a few private header files with project target membership
<img width="265" alt="screen shot 2017-06-21 at 2 57 31 pm" src="https://user-images.githubusercontent.com/28276156/27408543-16e568e2-5692-11e7-8c41-4232375a45ea.png">

- Moved `STPFile.m` into the `Models` group next to `STPFile.h`